vllm/sequence.py

```diff
@@ -57,6 +58,9 @@ class SequenceData:
     Attributes:
         prompt_token_ids: The token IDs of the prompt.
         output_token_ids: The token IDs of the output.
+        draft_token_ids: The token IDs proposed by the draft model, 
+                    it will be added by the SpecDecWorker when proposing draft tokens
+                    it will be cleared by the SpecDecWorker after accepting tokens
```

Comment:
Perhaps you can just refer the exact method name here.

Reply:
Then others will just goto that method to read the comments to figure out the detail if they want.